app/assets/components/ui-elements/oj-annotation.html

```diff
@@ -1,27 +1,41 @@
 <polymer-element name='oj-annotation' attributes="annotation color paper">
   <template>
     <div class='container' style=" background-color: {{color}}" >
-      <div class='content'>
-        <p>{{annotation.body}}</p>
-      </div>
 
-      <div class='comments'>
-        <template repeat="{{response in annotation.responses}}">
-          <p>{{response.body}}</p>
+      <div  on-tap="{{toggleCollapsed}}" class='content'>
+        <template if="{{collapsed}}">
+          <p layout horizontal>
+            <span> {{annotation.body | truncate}}</span>
+            <span flex></span>
+            <span>{{annotation.responses | count}} replies</span>
```

Comment:
Should be easy to give it a try. You want to give it a go and feedback?


—
Sent from Mailbox

On Tue, Feb 17, 2015 at 2:48 PM, Marc Rohloff <notifications@github.com>
wrote:

>>  
>> -      <div class='comments'>
>> -        <template repeat="{{response in annotation.responses}}">
>> -          <p>{{response.body}}</p>
>> +      <div  on-tap="{{toggleCollapsed}}" class='content'>
>> +        <template if="{{collapsed}}">
>> +          <p layout horizontal>
>> +            <span> {{annotation.body | truncate}}</span>
>> +            <span flex></span>
>> +            <span>{{annotation.responses | count}} replies</span>
> I had wondered that as well since data observing requires shimming in property readers and one would think that wouldn't be possible on a native type. On the other hand it is used as an example here https://www.polymer-project.org/docs/polymer/expressions.html#evaluating-expressions
> I guess it would just be nice to know if it works before we start doing ti all over.
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/openjournals/theoj/pull/123/files#r24852418